examples/olm/create-cr.yaml

```diff
@@ -15,4 +15,4 @@ spec:
       volumeMode: Filesystem
       fsType: xfs
       deviceNames:
-        - xvdf
+        - /dev/xvdf
```

Comment:
yeah we are. This was decided in conversation with CNV and OCS teams and internally openshift-storage team also kinda agreed that, this might not be a bad idea.

Reply:
Is the change reflected in docs? Are there any docs? :-)